### PR TITLE
[RNmobile] Improve "take video" icon

### PR DESCRIPTION
**Pull request by hypest**
## Description
This PR improves the "Take a Video" icon we're using on the media selection bottom sheet. So far, we've been using the "camera" icon, the very same as the "Take a Photo" icon, which is apparently less than ideal to be the same.

Screenshot 2020-03-16 at 15 26 23

## How has this been tested?
Use this gutenberg-mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/2024

To test:
1. Run the native mobile demo app and add a Media&Text block
2. Tap on the "Add image or video" button in the media area of the block
3. Notice the video-alt2 icon being used for the "Take a video" option

## Types of changes
1. Introducing the "video-alt2" icon, ported from the corresponding Dashicon https://github.com/WordPress/dashicons/blob/master/sources/svg/video-alt2.svg. Adding you @youknowriad as a reviewer on this PR for the web-related changes.
2. I'm not sure if I followed the proper procedure to port the icon. I copy/pasted the SVG path but I also moved the icon a bit to the right of its otherwise horizontal center, as it wasn't visually balanced in the very center. Used Inkscape for that movement. @jasmussen maybe you have some clue here whether this looks OK or not, or perhaps there's a better way of porting the icon from Dashicons.
3. Using the new icon on the native mobile bottom sheet when selecting media

## Checklist:
- [x] My code is tested.
- [x] My code follows the WordPress code style. 
- [x] My code follows the accessibility standards. 
- [x] My code has proper inline documentation. 
- [x] I've included developer documentation if appropriate. 
- [x] I've updated all React Native files affected by any refactorings/renamings in this PR.

### CODE REVIEW

File: packages/icons/src/index.js [hunk 1]
```diff
@@ -119,5 +119,6 @@ export { default as update } from './library/update';
 export { default as upload } from './library/upload';
 export { default as verse } from './library/verse';
 export { default as video } from './library/video';
+export { default as videoAlt2 } from './library/video-alt2';
```
Comment (youknowriad) [inline packages/icons/src/index.js]: That's probably not the best name for the icon as we're trying to be as descriptive as possible with the icons package. cc @jasmussen here

Comment (jasmussen) [reply packages/icons/src/index.js]: Yep. We need a new name and a new icon. I'll provide both when I have a chance.

**Review by ceyhun — APPROVED** [review]
Just tested on an Android 9 device and it looks good 👍 A snapshot test seems to be failing on Travis though

**Review by jasmussen — CHANGES_REQUESTED** [review]
I'll provide a new icon and a new name as soon as I can!

**Comment by jasmussen** [comment]
Can you try this icon?

```
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M16 9V7c0-.6-.4-1-1-1H4c-.6 0-1 .4-1 1v10c0 .6.4 1 1 1h11c.6 0 1-.4 1-1v-2l5 2.5v-11L16 9zm3.5 6.1l-5-2.5v3.9h-10v-9h10v3.9l5-2.5v6.2z"/></svg>
```

Screenshot 2020-03-16 at 16 42 49

Call it `capture-video` or `captureVideo`.

**Comment by pablohoneyhoney** [comment]
Here another version

icon-capture-video

[Figma](https://www.figma.com/file/fnyj380i05vGzuuH60frLQ/G2-Design?node-id=1970%3A3032)

**Comment by jasmussen** [comment]
Here's the SVG for Pablo's improved version:

```
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M14 6H5c-1.1 0-2 .9-2 2v8c0 1.1.9 2 2 2h9c1.1 0 2-.9 2-2V8c0-1.1-.9-2-2-2zm.5 10c0 .3-.2.5-.5.5H5c-.3 0-.5-.2-.5-.5V8c0-.3.2-.5.5-.5h9c.3 0 .5.2.5.5v8zm2.5-6v4l5 3V7l-5 3zm3.5 4.4l-2-1.2v-2.3l2-1.2v4.7z"/></svg>
```

**Comment by hypest** [comment]
Thanks for the new icon @jasmussen and @pablohoneyhoney !

Looks like the new icon has a different style than the rest on the sheet though (the new one is "hollow". See below.
screenshot-1584433634254

Can we have a "solid" version of the icon for this dialog? We could try to replace the other icons too but that starts to feel outside the scope of this PR.

**Comment by jasmussen** [comment]
> Looks like the new icon has a different style than the rest on the sheet though (the new one is "hollow". See below.

Honestly, I think the icons for "Choose from devic" and "Take a photo" are so classic looking that they are the ones standing out. 

If you like, I can provide two icons for those, and you can update them as well? Or you can do that in separate PRs?

**Comment by jasmussen** [comment]
The "Choose from device" can use the image icon, currently used for the Image block.

Here's a new icon for taking photos. Can call it `capture-photo` or `capturePhoto`:

```
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 8.1c-2.2 0-3.9 1.8-3.9 3.9s1.8 4 3.9 4 4-1.8 4-4-1.8-3.9-4-3.9zm0 6.4c-1.4 0-2.4-1.1-2.4-2.5s1.1-2.4 2.4-2.4 2.5 1.1 2.5 2.4-1.1 2.5-2.5 2.5zM20 5h-3.8l-1.7-2h-5L7.8 5H4c-1.1 0-2 .9-2 2v10c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V7c0-1.1-.9-2-2-2zm.5 12c0 .3-.2.5-.5.5H4c-.3 0-.5-.2-.5-.5V7c0-.3.2-.5.5-.5h4.5l1.7-2h3.6l1.7 2H20c.3 0 .5.2.5.5v10z"/></svg>
```

Screenshot 2020-03-17 at 14 49 24

**Comment by hypest** [comment]
Thanks @jasmussen ! Fair, let's replace the other icons too, in this PR. I'll use the capture-photo one and will also use the "choose from device" when you have it ready, thanks for the work here!

**Comment by iamthomasbishop** [comment]
Looks great, @jasmussen! Thank you!

**Comment by hypest** [comment]
Updated the capture-photo icon, here's how it looks:

screenshot-1584458740055

**Comment by jasmussen** [comment]
Need the image one as well :) It's in the Icon package, called `image`.

**Comment by hypest** [comment]
All updated now, here's how it looks:

screenshot-1584459468767

File: packages/icons/src/index.js (hunk 1 above)
Comment (hypest) [reply packages/icons/src/index.js]: Have changed the name with 654924b. Will mark this feedback as "Resolved" but please, feel free to comment more if needed, thanks!

**Review by jasmussen — APPROVED** [review]
Nice.

**Comment by pablohoneyhoney** [comment]
Here a new iteration to pair width and shapes a bit.

Screen Shot 2020-03-17 at 5 37 57 PM

**Comment by jasmussen** [comment]
Thanks for the vectors, Pablo! 

I updated the vectors (and the name, it was captureCamera instead of capturePhoto :) ), lets's see if the checks pass.

**Comment by hypest** [comment]
Thanks for the updates Pablo and Joen!

I went ahead and updated from master (and develop on https://github.com/wordpress-mobile/gutenberg-mobile/pull/2024) and all is looking good. I'll merge when CI gets green.

Here's how the bottom sheet looks like at the moment:

screenshot-1584541055892

**Pull request merged**